src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
```

Comment:
following BEM methodology make your star as an element of stars block. Apply to the same cases
```suggestion
      <div class="stars__star"></div>
```